packages/eslint-plugin/CHANGELOG.md

```diff
@@ -10,10 +15,9 @@
 - New Rule: [`@wordpress/i18n-no-collapsible-whitespace`](https://github.com/WordPress/gutenberg/blob/master/packages/eslint-plugin/docs/rules/i18n-no-collapsible-whitespace.md)
 - New Rule: [`@wordpress/i18n-ellipsis`](https://github.com/WordPress/gutenberg/blob/master/packages/eslint-plugin/docs/rules/i18n-ellipsis.md)
 
-### Breaking Changes
+### Bug Fixes
```

Comment:
So you moved this section up, it looked confusing at first 😃

Reply:
I contemplated to add a comment to my own pull request, then I worried it might not get a review if someone had seen there were comments already 😬

But yes, it was just to help emphasize Breaking Changes as first:

- Easier for publisher to know how to assign new version.
- Easier for consumer to know most impactful changes first and foremost.

Reply:
It appears you had the same idea in #21424 ! So the changes aren't as much here anymore 😄